### ISSUE TITLE
Accessibility: Envoy Missing a Translation Key

### ISSUE THREAD

**Issue by ImaTimekeeperReadingRaptor** [devs-immortal/The-Aether#792]
### Problem

When I started a new world to see the changed for the mod, I ran into an Envoy, and saw the Jade mod didn't find the translation key for the entity. So I looked through the `en_us.json` file and noticed that the key of `entity.paradise_lost.envoy` is completely missing, yet other things related to the Envoy were present, such as `"item.paradise_lost.envoy_spawn_egg": "Envoy Spawn Egg"` and `"subtitles.paradise_lost.entity.envoy.damage": "Envoy Chimes",`

![2024-05-04_18 29 52](https://github.com/devs-immortal/Paradise-Lost/assets/65588483/61b109c6-ddf4-4318-8fd5-d772020f578a)

### Suggestions (Optional)

This problem could be fixed by adding the translation keys to the lang files for the `entity.paradise_lost.envoy` entity

_This is more of a joke then anything below_
I'll do the English ones for you :)
`"entity.paradise_lost.envoy": "Envoy",`

### Mod Version

Beta 2.2.0 for 1.19.2
